src/copyright/ui/list.php

```diff
@@ -174,19 +166,17 @@ function GetRequestedRows($rows, $excl, &$NumRows, $filter)
       if (($filter == "nolic") and ($rf_clause))
       {
         /* discard file unless it has no license */
-        $sql = "select rf_fk from license_file where ($rf_clause) and pfile_fk={$row['pf']}";
-        $result = pg_query($PG_CONN, $sql);
-        DBCheckResult($result, $sql, __FILE__, __LINE__);
-        $FoundRows = pg_num_rows($result);
-        pg_free_result($result);
-        if ($FoundRows == 0)
+        $sql = "select rf_fk from license_file where ($1) and pfile_fk=$2";
```

Comment:
This won't work as expected, since `$1` will replaced by a *value* in execution.
This means `$1` will be replaced by true instead of the *expression*. Hence, also files with real licenses will be filtered out.
To test it: upload two files with identical copyright, one file with license, one file without license